app/views/departments/_contact_fields.slim

```diff
@@ -0,0 +1,36 @@
+= f.hidden_field :contactable_id
+= f.hidden_field :contactable_type
```

Comment:
you're right. They where rubble from trying to get the double nesting with polymorphic running. But arn't needed